src/Application/Command/GenerateIngredientsList.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\Command;
+
+use App\Domain\Ingredients\Dto\Ingredient;
+use App\Domain\Ingredients\UseCase\MealAndCocktailIngredientsList;
+use App\UserInterface\Output\ConsoleOutputPrinter;
+
+class GenerateIngredientsList
+{
+    private MealAndCocktailIngredientsList $useCase;
+    private ConsoleOutputPrinter $output;
+
+    public function __construct()
+    {
+        $this->useCase = new MealAndCocktailIngredientsList();
```

Comment:
Use dependency injection if you can for these properties

Reply:
Implemented with de8d9a3 commit